api/v3/handlers/events.go

```diff
@@ -0,0 +1,124 @@
+package handlers
```

Comment:
I would move this back to `openmeter/ingest/httpdriver`. The reason being is that I would avoid having this kitchen-sink package for handlers.

Once we rewrite all of our stuff to V3 this package will be quite huge and filled with helper and zillions of files and dependencies.

Events is a bad example as it's a single endpoint handler, but just imagine adding entitlements, subscriptions and billing here.